### PR TITLE
qmrestore: add German translation

### PR DESCRIPTION
**Pull request by Braintelligence**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
version 7.2, Wed 04 May 2022 07:30:00 AM CEST ( https://pve.proxmox.com/pve-docs/qmrestore.1.html )

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Note to maintainers: Review this PR after #8790 's merge.

**Comment by Braintelligence** [comment]
FYI #8790 was merged 💃

File: pages.de/linux/qmrestore.md
Comment (navarroaxel) [inline pages.de/linux/qmrestore.md:4]: We have an standard translation for this
```suggestion
> Weitere Informationen: <https://pve.proxmox.com/pve-docs/qmrestore.1.html>.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**
